LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Robocorp Technologies, Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
is this change on purpose?

Reply:
Yes, if the users use template as a base without looking at the file it is a bit weird if the license automatically states Robocorp